entrypoint.sh

```diff
@@ -1 +1,7 @@
-mdl ./
\ No newline at end of file
+output=$(mdl $INPUT_PATH)
+echo ::set-output name=output::"$output"
+if [ ! -z "$output" ]
+then
+    echo "FOUND MARKDOWN ISSUS"
```

Comment:
Will it output the actual errors still? Can you paste an example output here?

Reply:
https://github.com/steinbrueckri/gha-test/runs/430139684